spec/models/post_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Post do
+  subject { Post.new(title: 'Sunday', text: 'Sunny Sunday') }
+
+  before { subject.save }
+
+  it 'title should be Sunday' do
+    subject.title = 'Sunday'
+  end
+
+  it 'text should be Sunny Sunday' do
+    subject.text = 'Sunny Sunday'
```

Comment:
* According to the project [requirements ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.) 

> Add unit specs for all of your models' methods and validations.
* Kindly add unit specs for the `update_posts_counter` method 
* Kindly add unit specs for the `recent_comments` method 
* And write unit specs that cover all validations specified here 🙏